### DISCUSSION TITLE
Sensors: values and colour?

### DISCUSSION THREAD

**Discussion by janreinhardt** [nicolargo/glances#3038] [Q&A]
Please, why is the temp of these 3 sensors shown in different colour? Can I change it?

![IMG_1994](https://github.com/user-attachments/assets/aa81b073-8b98-4d1e-a1fa-a5dde8da65f9)

**Comment by janreinhardt** [comment]
Anyone? Why is the font colour white for all my Raspberry Pi 5 standard sensors, but green for the extra SSD / the addon HAT?

**Comment by nicolargo** [comment]
@janreinhardt by default Glances takes system thresholds.

Example on my system:

```
❯ sensors
nouveau-pci-2100
Adapter: PCI adapter
fan1:        1393 RPM
temp1:        +44.0°C  (high = +95.0°C, hyst =  +3.0°C)
                       (crit = +105.0°C, hyst =  +5.0°C)
                       (emerg = +135.0°C, hyst =  +5.0°C)

coretemp-isa-0000
Adapter: ISA adapter
Package id 0:  +30.0°C  (high = +93.0°C, crit = +103.0°C)
Core 0:        +30.0°C  (high = +93.0°C, crit = +103.0°C)
Core 1:        +29.0°C  (high = +93.0°C, crit = +103.0°C)
Core 3:        +30.0°C  (high = +93.0°C, crit = +103.0°C)
Core 4:        +29.0°C  (high = +93.0°C, crit = +103.0°C)
```

Check that system thresholds are avaible for Pi 5 standard sensors.

Note: it is also possible to overwrite system threshold from the Glances configuration file.
